wagtail/utils/urlpatterns.py

```diff
@@ -1,14 +1,18 @@
 from __future__ import absolute_import, unicode_literals
-
 from functools import update_wrapper
+from django import VERSION as DJANGO_VERSION
 
 
 def decorate_urlpatterns(urlpatterns, decorator):
     for pattern in urlpatterns:
         if hasattr(pattern, 'url_patterns'):
             decorate_urlpatterns(pattern.url_patterns, decorator)
 
-        if hasattr(pattern, '_callback'):
-            pattern._callback = update_wrapper(decorator(pattern.callback), pattern.callback)
+        if DJANGO_VERSION <= (1, 9):
+            if hasattr(pattern, '_callback'):
+                pattern._callback = update_wrapper(decorator(pattern.callback), pattern.callback)
+        else:
+            if hasattr(pattern, 'callback') and callable(pattern.callback):
```

Comment:
@timgraham Yep, the `getattr` version looks fine to me - we don't need to check `callable(pattern.callback)`, because _only_ callables are valid here as of Django 1.10 (and this is enforced by https://github.com/django/django/blob/ecb59cc6579402b68ddfd4499bf30edacf5963be/django/conf/urls/__init__.py#L82-L85). Thanks!

Changed accordingly in https://github.com/gasman/wagtail/commit/33fa4c3ef9a3b83359841fc714fd22426b7c0f39 - @pjstevns / @m1kola, does this change look OK to you?